bug_update.php

```diff
@@ -73,6 +73,19 @@
 	$g_project_override = $t_existing_bug->project_id;
 }
 
+if( gpc_get_int( 'vote', 0 ) ) {
+	if( access_user_can_vote_for_bug( $f_bug_id, $t_current_user_id ) ) {
+		$t_existing_bug->vote();
+		
+		if( access_has_bug_level( config_get( 'monitor_bug_threshold' ), $f_bug_id ) ) {
+			bug_monitor( $f_bug_id, $t_current_user_id );	
+		}
+	}
+	form_security_purge( 'bug_update' );
```

Comment:
I would suggest removing this functionality and stick with just automated crash reports for this PR.  Then we can add votes as a separate PR/feature.